features/bevragen/persoon/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,300 @@
+Functionaliteit: autorisatie verblijfplaatsgegevens Persoon
+
+    Achtergrond:
+      Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0599                              | 19860801                           |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | gemeente_code                | 0599            |
+      | straatnaam (11.10)           | Borgesiusstraat |
+      | huisnummer (11.20)           | 103             |
+      | huisletter (11.30)           | b               |
+      | huisnummertoevoeging (11.40) | 2               |
+      | postcode (11.60)             | 2497BV          |
+      | woonplaatsnaam (11.70)       | Scheveningen    |
+
+
+  Rule: Wanneer met fields gevraagd wordt om een veld waarvoor de gebruiker niet geautoriseerd is, wordt een foutmelding gegeven
+    Om een veld te mogen vragen moet de afnemer geautoriseerd zijn voor alle LO BRP rubrieken waar het veld mee gevuld of van afgeleid wordt
+
+    @fout-case
+    Abstract Scenario: Afnemer vraagt om een veld waarvoor deze niet geautoriseerd is
```

Comment:
`... om een veld waarvoor ...` --> `... om het <gevraagd veld> veld waarvoor ...`

Of nog beter:

`... om een veld waarvoor ...` --> `... om het <gba veld> veld waarvoor ...`
en de volgende kolom toevoegen aan de voorbeelden tabel:
```
| gba veld                                   |
| functie adres (10.10)                      |
| straatnaam (11.10)                         |
| huisnummer (11.20)                         |
| huisletter (11.30)                         |
| huisnummertoevoeging (11.40)               |
| aanduiding bij huisnummer (11.50)          |
| postcode (11.60)                           |
| woonplaatsnaam (11.70)                     |
| regel 1 adres buitenland (13.30)           |
| regel 2 adres buitenland (13.40)           |
| regel 3 adres buitenland (13.50)           |
| identificatiecode verblijfplaats (11.80)   |
| identificatiecode nummeraanduiding (11.90) |
| datum ingang geldigheid (85.10)            |
```

Reply:
scenario titel gewijzigd naar: Afnemer vraagt <gevraagd veld>, waarvoor deze niet geautoriseerd is